scripts/genDistance.sh

```diff
@@ -61,13 +61,14 @@ if [ $RESUME -le $STEP ]; then
     for binary in $(echo "$binaries"); do
 
       echo "($STEP) Constructing CG for $binary.."
-      while ! opt -dot-callgraph $binary.0.0.*.bc >/dev/null 2> $TMPDIR/step${STEP}.log ; do
+      prefix="$TMPDIR/dot-files/$(basename $binary)"
+      while ! opt -dot-callgraph $binary.0.0.*.bc -callgraph-dot-filename-prefix $prefix >/dev/null 2> $TMPDIR/step${STEP}.log ; do
```

Comment:
Does this address #87?

Reply:
No, it doesn't. The output directory and name of the created file have changed for the `-dot-callgraph` flag, the `-callgraph-dot-filename-prefix` flag makes sure, that the callgraph dot file still lands in the `dot-files` directory.

I'd have an idea how to address #87, but currently I don't have time. I'll pitch the idea in the issue report later on